### PR TITLE
Us/update contract erc20

### PR DESCRIPTION
**Pull request by ahamdi**
After changing the contract to be an ERC20 compliant with no specific feature inside it, the wallet application should have been updated to remove those functionalities from the app and update code to use the new contract.

### CODE REVIEW

File: wallet-webapps-admin/src/main/webapp/vue-app/components/admin/contracts/WalletAdminContractTab.vue [hunk 1]
```diff
@@ -16,29 +16,25 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
   <v-flex
-    v-if="contractDetails && contractDetails.title"
+    v-if="contractDetails && contractDetails"
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-admin/src/main/webapp/vue-app/components/admin/contracts/WalletAdminContractTab.vue]:
```suggestion
    v-if="contractDetails"
```

File: wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue [hunk 2]
```diff
@@ -406,8 +401,6 @@ export default {
       const walletTableHeaders = this.walletHeaders.slice();
       if (!this.contractDetails) {
         walletTableHeaders.splice(4, 1);
```
Comment (GouadriaHanen) [inline wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue:403]: walletTableHeaders.splice() repeated 2 times

File: wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue [hunk 3]
```diff
@@ -204,14 +202,14 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+                        <template v-else-if="!props.item.disabledUser && !props.item.deletedUser && props.item.enabled && (Number(props.item.etherBalance) === 0 || (etherAmount && walletUtils.toFixed(props.item.etherBalance) < Number(etherAmount)))">
                           <v-list-item :disabled="adminNotHavingEnoughEther" @mousedown="$event.preventDefault()">
                             <v-list-item-title class="options" @click="openSendEtherModal(props.item)">{{ $t('exoplatform.wallet.button.sendEther') }}</v-list-item-title>
                           </v-list-item>
                           <v-divider />
                         </template>
                         <v-list-item
-                          v-if="contractDetails && !contractDetails.isPaused && !props.item.disabledUser && !props.item.deletedUser && props.item.enabled && props.item.isApproved && tokenAmount > 0"
+                          v-if="contractDetails && !contractDetails.isPaused && !props.item.disabledUser && !props.item.deletedUser && props.item.enabled && tokenAmount > 0"
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue]:
```suggestion
                          v-if="!props.item.disabledUser && !props.item.deletedUser && props.item.enabled && tokenAmount > 0"
```

File: wallet-webapps-common/src/main/java/org/exoplatform/wallet/blockchain/service/EthereumBlockchainTransactionService.java [hunk 4]
```diff
@@ -54,75 +54,25 @@
...
-
-  private static final String              TRANSACTIONFEE_EVENT_HASH   =
-                                                                     EventEncoder.encode(ERTTokenV2.TRANSACTIONFEE_EVENT);
+                                                                  EventEncoder.encode(OWNERSHIPTRANSFERRED_EVENT);
+//
+//  private static final String              NOSUFFICIENTFUND_EVENT_HASH =
+//                                                                       EventEncoder.encode(MeedsToken.NOSUFFICIENTFUND_EVENT);
+//
+//  private static final String              TRANSACTIONFEE_EVENT_HASH   =
+//                                                                     EventEncoder.encode(MeedsToken.TRANSACTIONFEE_EVENT);
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-common/src/main/java/org/exoplatform/wallet/blockchain/service/EthereumBlockchainTransactionService.java]: to be deleted !

File: wallet-webapps-common/src/main/java/org/exoplatform/wallet/blockchain/service/EthereumWalletTokenAdminService.java [hunk 5]
```diff
@@ -288,44 +262,20 @@ public final TransactionDetail initialize(TransactionDetail transactionDetail, S
 
     setIssuer(transactionDetail, issuerUsername);
 
-    if (isInitializedAccount(receiverAddress)) {
+    //TODO this block is to reimplement
+/*    if (isInitializedAccount(transactionDetail.getToWallet())) {
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-common/src/main/java/org/exoplatform/wallet/blockchain/service/EthereumWalletTokenAdminService.java]: To be deleted !

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

File: wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue (hunk 2 above)
Comment (ahamdi) [reply wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue:403]: Yes, that was to remove two elements from the array when the contract is not loaded.

**Review by Rdidmohamedelhadi — APPROVED** [review]
LGTM

**Review by GouadriaHanen — APPROVED** [review]
LGTM

**Pull request merged**